describe.py

```diff
@@ -394,6 +394,7 @@ def document_api_from_discovery_document(uri):
   if FLAGS.discovery_uri:
     document_api_from_discovery_document(FLAGS.discovery_uri)
   else:
+    api_directory = {}
```

Comment:
This could be simplified using `colletions.defaultdict`, e.g.:

```
import collections
...
    api_directory = collecctions.defaultdict(list)
```

and then later just:

```python
    api_directory[api['name']].append(api['version'])